src/components/menu-right/DotTypeEditor.vue

```diff
@@ -104,4 +131,11 @@ export default Vue.extend({
 });
 </script>
 
-<style scoped lang="scss"></style>
+<style scoped lang="scss">
+.menu-right-dot-appearance {
+  vertical-align: middle;
+  float: right;
+  width: 10%;
+  height: 10%;
```

Comment:
nit: Instead of percentages for the sizing, why not use a fixed width and height?